internal/core/Cargo.toml

```diff
@@ -60,7 +60,7 @@ lyon_geom = { version = "1.0", optional = true  }
 lyon_path = { version = "1.0" , optional = true }
 lyon_extra = { version = "1.0.1", optional = true }
 num-traits = { version = "0.2", default-features = false }
-once_cell = { version = "1.5", default-features = false }
+once_cell = { version = "1.5", default-features = false, features = ["atomic-polyfill"] }
```

Comment:
I'm curious, where does that come from?

Reply:
We need that feature to build on hardware that don't have atomic.
The mcu-board-support enable this feature as well, but i want to be able to compile without dirrectly linking against once_cell